### ISSUE TITLE
Set up an Open Collective

### ISSUE THREAD

**Issue by voxpelli** [neostandard/neostandard#21]
To make sure `neostandard` become a collective and sustainable effort I think we should set up an Open Collective.

Thoughts @mcollina, @bcomnes, @wesleytodd?

**Comment by voxpelli** [comment]
@wesleytodd Would be a great collaboration! I'm thinking of basing the mocha ones on the ones that ESLint has. Maybe we can make a generic boilerplate version that we all can adopt.

**Comment by wesleytodd** [comment]
I was also going to propose we follow eslint's setup on express! This sounds great.

**Comment by voxpelli** [comment]
Was suggested by @tobie to bring it to the OpenJSF CPC / a collab space

**Comment by tobie** [comment]
Yes please! Let's build resources that benefit all projects, here!
